README.md

```diff
@@ -162,6 +162,8 @@ require('telescope').setup{
     file_previewer = require'telescope.previewers'.cat.new, -- For buffer previewer use `require'telescope.previewers'.vim_buffer_cat.new`
     grep_previewer = require'telescope.previewers'.vimgrep.new, -- For buffer previewer use `require'telescope.previewers'.vim_buffer_vimgrep.new`
     qflist_previewer = require'telescope.previewers'.qflist.new, -- For buffer previewer use `require'telescope.previewers'.vim_buffer_qflist.new`
+    buffer_previewer_wrap = false,
+    buffer_previewer_maker = require'telescope.previewers'.buffer_previewer_maker -- Don't touch this one unless you want a bad time. We will do something with this and an extension
```

Comment:
maybe we should put a separate section `For Developers` so that we can put something like this here.